src/main/kotlin/org/rust/cargo/toolchain/Grcov.kt

```diff
@@ -18,6 +18,7 @@ class Grcov(private val grcovExecutable: Path) {
             .withParameters(
                 ProjectLayout.target,
                 "-t", "lcov",
+                "--llvm",
```

Comment:
Is this option a new one, i.e. does `0.4.3` version contain it? If no then we should update our minimal version in the corresponding notification

Reply:
Yes, `0.4.3` contains it.